dev_guide/cron_jobs.adoc

```diff
@@ -69,25 +49,64 @@ kind: CronJob
     spec:
       template:
         metadata:
-          labels:          <3>
+          labels:               <8>
             parent: "cronjobpi"
         spec:
           containers:
           - name: pi
             image: perl
             command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
-          restartPolicy: OnFailure <4>
+          restartPolicy: OnFailure <9>
 ----
 
-1. Schedule for the job. In this example, the job will run every minute.
-2. Job template. This is similar to the xref:../dev_guide/jobs.adoc#creating-a-job[job example].
-3. Sets a label for jobs spawned by this cron job.
-4. The restart policy of the pod. This does not apply to the job controller. See xref:../dev_guide/jobs.adoc#creating-a-job-known-issues[Known Issues and Limitations] for details.
+<1> The schedule field for the job is specified in 
+    link:https://en.wikipedia.org/wiki/Cron[cron format]. In this example, the
+    job runs every minute.
+<2> The concurrency policy specifies how to treat concurrent jobs
+    within a cron job. Only one of the following policies can be specified. By
+    default, the policy is `Allow`.
+    * `Allow` allows multiple instances of the job to run concurrently.
+    * `Forbid`: prevents multiple instances of the job from running concurrently.
+      Scheduled runs are skipped if the previous run has not finished.
+    * `Replace`: cancels a currently running instance of the job and replaces
+    the job with a new instance.
+<3> The starting deadline specifies a deadline (in seconds) for starting the job
+    if a scheduled run is missed for any reason. After the deadline, the cron job
+    does not start the job. Missed job executions are counted as failed jobs. By
+    default, jobs have no deadline. Be aware that when this field is set, the cron job
+    controller counts the number of missed jobs that occur in the interval between
+    the value for the deadline and the current moment. For example, if
+    `startingDeadlineSeconds` is set to `200`, the controller counts the number
+    of missed jobs in the last 200 seconds. For more information, see the limitations
+    for link:https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/[cron jobs]
+    in the Kubernetes documentation for cron job concepts.
+<4> The suspend field is used to prevent subsequent executions of the cron job.
+    If set to `true`, then all subsequent executions are prevented from starting.
+    By default, the value is `false` and jobs are executed.
+<5> The successful jobs history limit specifies the number of finished jobs to
+    retain. By default, `3` jobs are retained.
+<6> The failed jobs history limit specifies the number of failed jobs to retain.
+    By default, `1` job is retained.
+<7> The job template specifies the job to run. The field is similar to the
+    xref:../dev_guide/jobs.adoc#creating-a-job[job example].
+<8> The labels field specifies a label to set for jobs that are started by
```

Comment:
Are all of these values required? The description of this one makes it sounds optional. If you know which ones are required and optional, I suggest labeling the optional ones like "<n> Optional:"